cluster/gke/util.sh

```diff
@@ -181,6 +181,11 @@ function kube-up() {
 
   # Bring up the cluster.
   "${GCLOUD}" ${CMD_GROUP:-} container clusters create "${CLUSTER_NAME}" "${create_args[@]}"
+
+  if [[ ! -z "${HEAPSTER_MACHINE:-}" ]]; then
```

Comment:
Context: we can't fit heapster into n1-standard-1 machines in large clusters, and I would like to avoid having all large machines to test heapster.